.github/workflows/lint.yml

```diff
@@ -0,0 +1,71 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  checks: write
+  contents: write
+
+jobs:
+  lint-python:
+    name: Lint Python
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check changed py files
+        id: changed-files
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            **/*.py
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        with:
+          python-version: 3.8
+
+      - name: Install Python dependencies
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        run: pip install flake8
+
+      - name: Run flake8 linter
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
+        uses: wearerequired/lint-action@v2
+        with:
```

Comment:
I assume this auto-discovers the `.flake8` config file (and likewise for javascript)?

Reply:
Correct.